depends/packages/qt.mk

```diff
@@ -1,53 +1,114 @@
 PACKAGE=qt
-$(package)_version=5.2.1
-$(package)_download_path=http://download.qt-project.org/official_releases/qt/5.2/$($(package)_version)/single
-$(package)_file_name=$(package)-everywhere-opensource-src-$($(package)_version).tar.gz
-$(package)_sha256_hash=84e924181d4ad6db00239d87250cc89868484a14841f77fb85ab1f1dbdcd7da1
+$(package)_version=5.4.2
+$(package)_download_path=http://download.qt.io/official_releases/qt/5.4/$($(package)_version)/submodules
+qt_suffix=opensource-src-$($(package)_version).tar.gz
+$(package)_file_name=qt5-$(qt_suffix)
+$(package)_sha256_hash=165840bcb69f670070cc71247179b696a839917b0758f332b88b6aab45fac400
 $(package)_dependencies=openssl
 $(package)_linux_dependencies=freetype fontconfig dbus libxcb libX11 xproto libXext
 $(package)_build_subdir=qtbase
 $(package)_qt_libs=corelib network widgets gui plugins testlib
-$(package)_patches=mac-qmake.conf fix-xcb-include-order.patch qt5-tablet-osx.patch qt5-yosemite.patch
+$(package)_patches=mac-qmake.conf fix-xcb-include-order.patch
+qtbase_file_name=qtbase-$(qt_suffix)
```

Comment:
Ah, I figured there was some reason like that... at least it will help consistency, but I'd note that $(package) in this case is "qt" ;)